CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 ## v0.44.0
 
+## 🧰 Bug fixes 🧰
```

Comment:
v0.44.0 was released already, shouldn't this be under the "Unreleased" part?

Reply:
Yup, that's a fun off-by-one error!  Fixed.